common/tensorrt_common/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ find_package(TENSORRT)
 
 if(NOT (CUDAToolkit_FOUND AND CUDNN_FOUND AND TENSORRT_FOUND))
   message(WARNING "cuda, cudnn, tensorrt libraries are not found")
+  ament_package()
```

Comment:
I am using ament_package() instead of ament_auto_packages() to match with L82 in the original file.